dm/syncer/syncer.go

```diff
@@ -2442,6 +2442,13 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 			if needContinue {
 				continue
 			}
+		case *replication.TableMapEvent:
+			s.tctx.L().Info("unhandled TableMap event",
+				zap.String("schema", string(ev.Schema)),
+				zap.String("table", string(ev.Table)),
+			)
```

Comment:
```suggestion
```

The "unhandled event" is used to detect some new events of future MySQL/MariaDB versions (I guess). So for these old type of events we don't need to log them, they are handled as no-op.

BTW we may also need to change line 2438

Reply:
@lance6716 The suggestion in your comment doesn't seem to change anything. Did you intent to suggest a code change?

I made changes based on your suggestion in ecc9985a47d63ed82d4bd059af4908ca9ba122ca

Reply:
I mean we don't need to print log for these two types of events. Just move them out of `default` case

Reply:
Done in 5773c89d79b2ee8598414018ce1a11f975012bb0